tools/xcodeproj_shims/xcodeproj-installer.sh

```diff
@@ -6,10 +6,21 @@ readonly project_path="${PWD}/$(project_short_path)"
 readonly dest="${BUILD_WORKSPACE_DIRECTORY}/$(project_short_path)/"
 readonly tmp_dest=$(mktemp -d)/$(project_full_path)/
 
-readonly stubs_dir="${dest}/bazelstubs"
+readonly installer="$(installer_short_path)"
+
+rm -fr "${tmp_dest}"
+mkdir -p "$(dirname $tmp_dest)"
+cp -r "${project_path}" "$tmp_dest"
+chmod -R +w "${tmp_dest}"
+
+readonly stubs_dir="${tmp_dest}/bazelstubs"
```

Comment:
installing the stubs into the tmp dir means they will get chmod'd, allowing them to be overwritten by subsequent invocations

Reply:
cc @amberdixon this is the change I showed you on Tuesday